packages/server/src/seeder/utils.ts

```diff
@@ -13,13 +16,17 @@ export async function markdownToJSON() {
 /**
  * Parse the Markdown to HTML
  */
-export async function markdownToHTML() {
-    // Markdown file is located in the assets folder
-    // markdown file is a article about the cat journey to the moon.
-    // read the markdown and convert it to HTML
-    // return the actual HTML string as { HTML: string }
-    // comment the library used for this purpose in the related issue on GitHub
-    // if you can write the pure TypeScript function for this purpose then awesome . :) You are really awesome !
+// used showdown (https://www.npmjs.com/package/showdown)
+export async function markdownToHTML(inputPath: string) {
```

Comment:
You should not accept any arg. Since the file will always be located in the `src/assets` folder.